eclib/JWord.ec

```diff
@@ -2224,8 +2224,10 @@ abstract theory W_WS.
    op VPMUL_'Ru'S (w1 : WB.t) (w2 : WB.t) =
      map2 WS.( * ) w1 w2. 
 
+(* conflict with JModel
```

Comment:
Maybe the good patch is to fix this definition and then to remove the definition in JModel.
Also the definition VPMUL_'Ru'S should be renamed into VPMULL_'Ru'S